src/Illuminate/Support/helpers.php

```diff
@@ -261,11 +261,21 @@ function retry($times, callable $callback, $sleepMilliseconds = 0, $when = null)
     /**
      * Get a new stringable object from the given string.
      *
-     * @param  string  $string
-     * @return \Illuminate\Support\Stringable
+     * @param  string|null  $string
+     * @return \Illuminate\Support\Stringable|mixed
      */
-    function str($string)
+    function str($string = null)
     {
+        if (! isset($string)) {
```

Comment:
This should be `if (func_num_args() == 0) {` - as null is equivalent to `''` when coerced to string:

```php
// str($post->title)->isEmpty();
// etc...
```

Reply:
Right, didn't think of that. Will fix that in a moment.